chain-api/src/types/DexDtos.ts

```diff
@@ -660,7 +656,6 @@ export class GetRemoveLiqEstimationResDto extends ChainCallDTO {
 export class CollectProtocolFeesResDto extends ChainCallDTO {
   @IsNotEmpty()
   @BigNumberProperty()
```

Comment:
```suggestion

  @BigNumberProperty()
```

I don't recommend combining `@IsNotEmpty` with BigNumber related decorators

Reply:
Since these inputs are expected to accept a wide range of numbers, we have opted not to include additional decorators. Instead, we have simply removed the `@IsNotEmpty` decorator.